### DISCUSSION TITLE
Why does taking the values of the H-Field at one point along the x-axis look like this?

### DISCUSSION THREAD

**Discussion by feldnerd** [magpylib/magpylib#765] [Q&A] [answered]
Hi magpylib community,

I am interested in what the H-Field is doing on a certain line (dashed line in the image). If I make a bar chart out of the H-Field at this dashed line, it mostly looks as expected. There are positive regions and negative regions depending on where magnets are and their polarization. However, I am unsure why there are gaps. It might just be an artefact of the plotting, but just wanted to check if it is something to do with the H-Field?

Creating the magnets was simply a modification of the [Computing the Field](https://magpylib.readthedocs.io/en/latest/_pages/gallery/gallery_tutorial_field_computation.html#gallery-tutorial-field-computation) tutorial. And plotting uses this code `plt.barh(width=H[:,700,0], y = np.linspace(0,1000,1000))`

<img width="769" alt="magpylib_check" src="https://github.com/magpylib/magpylib/assets/56836977/66e122d6-0e66-461c-b4a8-3326249743d3">

**Comment by OrtnerMichael** [comment, marked as answer]
this is a `barh` graphic "problem" .... its not a really a problem its designed to do this. if you want a continuous line you can you `plt.plot()` :)

**Reply by feldnerd** [reply]
Ah I see, thanks!
